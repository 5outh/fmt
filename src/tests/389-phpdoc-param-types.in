<?php
//passes: GeneratePHPDoc
class Tools {

}

class Test {
	/**
	 * @var float
	 */
	private $f = 1.25;

	public function __construct(\Traversable $t, Tools $k, array $a, $b) {
		$this->a = $a;
		$this->b = $b;
	}

	public function getF($t){
		return $this->f . $t;
	}
}