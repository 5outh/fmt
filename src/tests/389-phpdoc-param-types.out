<?php
//passes: GeneratePHPDoc
class Tools {

}

class Test {
	/**
	 * @var float
	 */
	private $f = 1.25;

	/**
	 * @param \Traversable $t
	 * @param Tools $k
	 * @param array $a
	 * @param $b
	 */
	public function __construct(\Traversable $t, Tools $k, array $a, $b) {
		$this->a = $a;
		$this->b = $b;
	}

	/**
	 * @param $t
	 * @return mixed
	 */
	public function getF($t) {
		return $this->f . $t;
	}
}